judgments/tests/tests.py

```diff
@@ -206,6 +209,18 @@ def test_judgment_results(self, fake_result, fake_advanced_search):
             response, '<meta name="robots" content="noindex,nofollow" />'
         )
 
+    @patch("judgments.views.results.perform_advanced_search")
+    @patch("judgments.models.SearchResult.create_from_node")
+    def test_static_page(self, fake_result, fake_advanced_search):
```

Comment:
Should a static page have `noindex,nofollow`?

Reply:
No, it is safe to index; this tests that it should not.